Source/WebCore/html/HTMLMetaElement.h

```diff
@@ -50,11 +50,15 @@ class HTMLMetaElement final : public HTMLElement {
     void didFinishInsertingNode();
     void removedFromAncestor(RemovalType, ContainerNode&) final;
 
-    void process();
+    void process(const AtomString* oldValue = nullptr);
 
     std::optional<MQ::MediaQueryList> m_mediaQueryList;
 
     std::optional<Color> m_contentColor;
+
+#if ENABLE(DARK_MODE_CSS)
+    bool isNameColorScheme(const AtomString& nameValue);
```

Comment:
Can be marked as `static`.

Reply:
You don't even need to declare it in the header, having the static function at the top of the cpp file would be better here I think. If you insist on keeping it in the header, at least move it *before* the data members. Data members always come last, per our usual style.

Reply:
Thanks — marked `isNameColorScheme()` as `static` and moved it to the top of the cpp file